src/main/java/racingcar/controller/RacingCarController.java

```diff
@@ -0,0 +1,56 @@
+package racingcar.controller;
+
+import racingcar.domain.dto.CarsDTO;
+import racingcar.domain.model.Cars;
+import racingcar.service.GameService;
+import racingcar.view.InputView;
+import racingcar.view.OutputView;
+
+import java.util.List;
+
+public class RacingCarController {
+    private final InputView inputView;
+    private final OutputView outputView;
+    private final GameService gameService;
+    private Cars cars;
+    private int round;
+
+    public RacingCarController() {
+        this.inputView = new InputView();
+        this.outputView = new OutputView();
+        this.gameService = new GameService();
+    }
+
+    public void run() {
+        inputCarNames();
+        inputRoundCount();
+        startGame();
+        printWinners();
+    }
+
+    // 자동차 이름을 입력받아 Cars 객체 생성
+    private void inputCarNames() {
+        List<String> carNames = inputView.inputCarNames();
+        cars = new CarsDTO(carNames).getCars();
+    }
+
+    // 라운드 수 입력받기
+    private void inputRoundCount() {
+        round = inputView.inputRound();
+    }
+
+    // 게임 시작 및 결과 출력
+    private void startGame() {
+        outputView.printResultNotice();
+
+        for (int i = 0; i < round; i++) {
+            gameService.startRound(cars); // 각 라운드 진행
+            outputView.printRoundResult(cars.getCars()); // 각 라운드 결과 출력
+        }
+    }
+
+    // 우승자 계산 및 출력
+    private void printWinners() {
+        outputView.printWinners(gameService.chooseWinners(cars)); // 우승자 출력
+    }
```

Comment:
한 줄로 끝내는 방법과 아래와 같이 단계를 나누어 처리하는 방법 중 어떤 게 더 낫다고 생각하시나요?
```suggestion
// 우승자 계산 및 출력
private void printWinners() {
    List<Car> winners = gameService.chooseWinners(cars);
    outputView.printWinners(winners); // 우승자 출력
}
```

Reply:
원래는 규원님의 코드처럼 두 줄이였다가 간단한 로직이라 한 줄로 변경한 코드인데,

`확장성`의 측면에서 바라본다면, 
`단계를 나누어서 작성하는 것`이 추후에 winners 값을 더 가공하거나 추가 로직이 필요할 때 
중간 변수 사용이 유리하다고 생각하기 때문에 `단계를 나누어 처리하는 방법`이 더 나은 것 같네요!

@digitpic 님은 어떻게 생각하시나요?

Reply:
저 또한 한 라인에 `.` 이 여러 개가 포함되는 경우를 피하려고 합니다
동의합니다